### PR TITLE
First basic example and general format

### PR DESCRIPTION
**Pull request by nicolaferraro**
Wdyt?

### CODE REVIEW

File: 01-basic/readme.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
+# Camel K Basic Example
+ 
+![Camel K CI](https://github.com/openshift-integration/camel-k-example-basic/workflows/Camel%20K%20CI/badge.svg)
```
Comment (davsclaus) [inline 01-basic/readme.md:3]: This url for badge is likely wrong after migrating here

File: 01-basic/readme.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+Instructions are based on [VSCode Didact](https://github.com/redhat-developer/vscode-didact), so make sure it's installed
+from the VSCode extensions marketplace.
+
+From the VSCode UI, right-click on the `readme.didact.md` file and select "Didact: Start Didact tutorial from File". A new Didact tab will be opened in VS Code.
+
+Make sure you've opened this readme file with Didact before jumping to the next section.
+
+## Preparing the cluster
+
+This example can be run on any OpenShift 4.3+ cluster or a local development instance (such as [CRC](https://github.com/code-ready/crc)). Ensure that you have a cluster available and login to it using the OpenShift `oc` command line tool.
```
Comment (davsclaus) [inline 01-basic/readme.md:21]: What about plain Kubernetes?

File: 01-basic/readme.md [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+You need to create a new project named `camel-basic` for running this example. This can be done directly from the OpenShift web console or by executing the command `oc new-project camel-basic` on a terminal window.
+
+You need to install the Camel K operator in the `camel-basic` project. To do so, go to the OpenShift 4.x web console, login with a cluster admin account and use the OperatorHub menu item on the left to find and install **"Red Hat Integration - Camel K"**. You will be given the option to install it globally on the cluster or on a specific namespace.
+If using a specific namespace, make sure you select the `camel-basic` project from the dropdown list.
+This completes the installation of the Camel K operator (it may take a couple of minutes).
+
+When the operator is installed, from the OpenShift Help menu ("?") at the top of the WebConsole, you can access the "Command Line Tools" page, where you can download the **"kamel"** CLI, that is required for running this example. The CLI must be installed in your system path.
+
+Refer to the **"Red Hat Integration - Camel K"** documentation for a more detailed explanation of the installation steps for the operator and the CLI.
```
Comment (davsclaus) [inline 01-basic/readme.md:31]: Should be removed or refer to something at apache

Comment (oscerd) [reply 01-basic/readme.md:31]: +1, here we may need to point to the camel website, we can improve the documentation for sure.

**Comment by nicolaferraro** [comment]
My bad, I forgot to remove references after copy/paste..

**Comment by nicolaferraro** [comment]
Oh, I got what happened @davsclaus , @oscerd . The readme.md file was a symlink to readme.didact.md, but when I copied it from the original repo, it was transformed into a file, with the original comment.

Restoring the symlink..

**Comment by nicolaferraro** [comment]
How does it look @oscerd ?

**Pull request merged**